jsonschema-validation.xml

```diff
@@ -664,19 +650,21 @@
 
             <section title="not">
                 <t>
-                    This keyword's value MUST be an object. This object MUST be a valid JSON
-                    Schema.
+                    This keyword's value MUST be an object.
```

Comment:
Same thing:

This keyword's value MUST be a valid JSON Schema.

Of course, `{"not": true}` doesn't make much sense, I think it should be syntactically valid.